app/views/layouts/_nav.html.erb

```diff
@@ -47,8 +47,14 @@
         <div class="dropdown-menu dropdown-menu-end dropdown-menu-dark position-absolute shadow pb-2 text-center m-0" aria-labelledby="nav-user-dropdown">
           <div class="h2 py-3 px-4 text-nowrap"><%= current_user_name %></div>
           <%= link_to t(".id"), current_user_id_profile, class: "btn btn-primary-light mb-2" %>
-          <%= link_to "Reload", main_app.refresh_path, data: {turbo: false}, class: "dropdown-item text-primary-light" %>
-          <%= link_to "Logout", main_app.logout_path, data: {turbo: false}, class: "dropdown-item text-primary-light" %>
+          <% if current_user_admin? %>
+             <%= link_to t(".admin"), admin_root_path, class: "dropdown-item text-primary-light" %>
+             <button type="button" class="dropdown-item text-primary-light" data-bs-toggle="modal" data-bs-target="#wildcard-modal">
```

Comment:
nice, should it have a different style, or separate into a new section b/c it's special and admin only?

Reply:
Hmm... let's see what folks think when they try this a bit.  Since neither of these buttons does anything immediately when you click them (wildcard access just opens a model) - it wouldn't be disastrous if someone clicked it accidentally.